### PR TITLE
integrate 3 pods: SwiftLint, Localize.Swift, R.Swift

### PR DESCRIPTION
**Pull request by virustyt**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+platform :ios, '13'
+
+target 'FinanceAdvisor' do
+
+  use_frameworks!
+
+  pod 'SwiftLint'
+  pod 'Localize-Swift', '~> 3.2' #, :git => 'https://github.com/CocoaPods/Specs.git'
```
Comment (pandafliesinspace) [inline Podfile:10]: specs repo is already defined at the first line,  #, :git => 'https://github.com/CocoaPods/Specs.git' can be ommited.

**Comment by pandafliesinspace** [comment]
Please add SwiftLint script to project's build phase
image

**Review by pandafliesinspace — APPROVED** [review]
Can be merged after fixes

File: Podfile (hunk 1 above)
Comment (virustyt) [reply Podfile:10]: remove commented part with link

**Comment by virustyt** [comment]
remove commited string from pod file and add SwiftLint script

**Pull request merged**